cluster/gce/util.sh

```diff
@@ -45,10 +45,13 @@ if [[ "${MASTER_OS_DISTRIBUTION}" == "gci" ]]; then
 elif [[ "${MASTER_OS_DISTRIBUTION}" == "debian" ]]; then
   MASTER_IMAGE=${KUBE_GCE_MASTER_IMAGE:-${CVM_VERSION}}
   MASTER_IMAGE_PROJECT=${KUBE_GCE_MASTER_PROJECT:-google-containers}
+elif [[ "${MASTER_OS_DISTRIBUTION}" == "ubuntu" ]]; then
+  MASTER_IMAGE=${KUBE_GCE_MASTER_IMAGE}
```

Comment:
This isn't necessary, since it happens in `config-default.sh` already: https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/config-default.sh#L64